recording-daemon/rtpengine-recording.pod

```diff
@@ -259,6 +259,10 @@ input is mono audio, then the mixed output file would contain 4 audio channels.
 This mixing method requires an output file format which supports these kinds of
 multi-channel audio formats (e.g. B<wav>).
 
+=item B<--mix_num_input=>I<INT>
```

Comment:
Missed an `s`  here :)

Otherwise LGTM

Reply:
Actually the option also uses dashes `-` instead of underscores `_`

Reply:
fixed, thank you